pyknit/pyknit.py

```diff
@@ -11,15 +11,16 @@
 import math
 from typing import Set
 from pyknit import GaugeSwatch, Chart
+from pydantic import validate_arguments, PositiveInt
 
 VERSION = "pyKnit 0.0.3"
```

Comment:
hah, apparently I have too many version strings and missed one.  I'll make a separate commit to fix that.